cmd/bosun/expr/funcs.go

```diff
@@ -355,7 +364,7 @@ func parseGraphiteResponse(req *graphite.Request, s *graphite.Response, formatTa
 		// build tag set
 		nodes := strings.Split(res.Target, ".")
 		if len(nodes) < len(formatTags) {
-			return nil, fmt.Errorf(`returned target "%s" does not match format "%s"`, res.Target, formatTags)
+			return nil, graphite.NewError(req.String(), "", fmt.Errorf(`returned target "%s" does not match format "%s"`, res.Target, formatTags))
```

Comment:
parseGraphiteResponse should not return a graphite.Error because those are limited to web errors, not parse.